### ISSUE TITLE
logical-termination-point for MATR is not correct

### ISSUE THREAD

**Issue by schiavi-siae** [openBackhaul/MacAddressTableRecorder#114]
In file MacAddressRecorder-config.yaml, the "logical-termination-point": "matr-1-0-0-http-c-odl-4-0-2-000" is not correct.
The shouldn't be something with "..-op-..."? 

             {
                "uuid": "matr-1-0-0-op-fc-is-022",
                "name": [
                  {
                    "value-name": "ForwardingKind",
                    "value": "core-model-1-4:FORWARDING_KIND_TYPE_INVARIANT_PROCESS_SNIPPET"
                  },
                  {
                    "value-name": "ForwardingName",
                    "value": "PromptForUpdatingMacTableFromDeviceCausesMacTableBeingRetrievedFromDevice"
                  }
                ],
                "fc-port": [
                  {
                    "local-id": "100",
                    "port-direction": "core-model-1-4:PORT_DIRECTION_TYPE_INPUT",
                    "logical-termination-point": "matr-1-0-0-op-s-is-002"
                  },
                  {
                    "local-id": "200",
                    "port-direction": "core-model-1-4:PORT_DIRECTION_TYPE_OUTPUT",
                    **"logical-termination-point": "matr-1-0-0-http-c-odl-4-0-2-000"**
                  }
                ]
              },

**Comment by kmohr-soprasteria** [comment]
the correct uuid needs to be corrected to "matr-1-0-0-op-c-is-odl-4-0-2-000"
as this is what is also specified in the serviceList:
```
  - http-client:
      application-name: OpenDayLight
      release-number: 4.0.2
      uuid: matr-1-0-0-http-c-odl-4-0-2-000
...    
individual:
          - operation-name: /rests/operations/network-topology:network-topology/topology=topology-netconf/node={mount-name}/yang-ext:mount/mac-fd-1-0:provide-learned-mac-addresses
            uuid: matr-1-0-0-op-c-is-odl-4-0-2-000
```

**Comment by schiavi-siae** [comment]
Closed with issue #92
